articles/connections/passwordless/ios-touch-id-objc.md

```diff
@@ -12,6 +12,8 @@ languages:
 
 ::: warning
 This feature is disabled for new tenants as of June 8th 2017. Any tenant created after that date won't have the necessary legacy [grant types](/clients/client-grant-types) to use Touch ID. This document is offered as reference for older implementations.
+
+Please take a look at an alternative approach to [Touch ID Authentication](/libraries/lock-ios/v2/touchid-authentication).
```

Comment:
@jeffreylees this might look nicer broken out into `::: note` vs a `::: warning`.  Your call, team docs 😄 Same goes for the other file.